pkg/batchd/cache/resource_info.go

```diff
@@ -26,19 +26,26 @@ import (
 type Resource struct {
 	MilliCPU float64
 	Memory   float64
+	GPU      int64
 }
 
+const (
+	GPUResourceName = "alpha.kubernetes.io/nvidia-gpu"
```

Comment:
GPU resource name has been changed from v1.8 onwards. ( https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/#v18-onwards )
You should use ` nvidia.com/gpu` for v1.8 and v1.9.

Reply:
I found a useful const in the k8s api package: https://github.com/kubernetes/api/blob/master/core/v1/types.go#L4084
Probably using it would be the most straightforward way for the purpose?

Reply:
After v1.8, k8s delegated the management of GPU to `Nvidia device plugin`.
So the resource name is defined in `Nvidia device plugin`. https://github.com/NVIDIA/k8s-device-plugin/blob/66a35b71ac4b5cbfb04714678b548bd77e5ba719/server.go#L20

Reply:
Unfortunately the const seems to be private. We need to have our own local copy in kube-batchd...